{{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/common/main.tf

```diff
@@ -19,10 +19,16 @@ resource "databricks_secret" "cd_credentials_token" {
   scope        = databricks_secret_scope.cd_credentials.id
 }
 
+resource "databricks_secret" "cd_github_server" {
```

Comment:
Wait, is the cd_github_server a databricks secret? with a scope?  It's not a credential, right?

Reply:
(Maybe I'm missing something.)

Reply:
I am following the same style for `github_repo` [here](https://src.dev.databricks.com/databricks/mlops-stack/-/blob/%7B%7Bcookiecutter.project_name%7D%7D/.mlops-setup-scripts/cicd/common/main.tf?L22). I think the reason is to facilitate retrieving the value through dbutils for example [here](https://src.dev.databricks.com/databricks/mlops-stack/-/blob/%7B%7Bcookiecutter.project_name%7D%7D/notebooks/TriggerModelDeploy.py?L49).

Reply:
Ack, please double check this w/ @arpitjasa-db .

Reply:
Yeah we could consider instead writing out to a `json` output for example and pulling the value in from there for non-sensitive values, but this is fine for now since it's following the preset convention

Reply:
The issue is that it's misleading - for instance the scope value does not apply, but it's present. Can we at least document this and then follow-up (don't need to block this PR on it).